server/src/utils.ts

```diff
@@ -203,4 +203,13 @@ export function nLog(message: string, settings: NavigatorSettings){
     }
 }
 
+export function getPerlimportsProfile (settings: NavigatorSettings): string[] {
+    let profileCmd: string[] = [];
```

Comment:
We can use `const` here. It only prevents reassignment, not mutation.

```suggestion
    const profileCmd: string[] = [];
```